editor/test/integration/save_test.clj

```diff
@@ -31,298 +29,28 @@
-                 "**/broken_gui_resources.gui"
-                 "**/breaks_gui_resources.gui"
-                 "**/uses_gui_resources.gui"
-                 "**/uses_gui_resources_missing_files.gui"
-                 "**/uses_broken_gui_resources.gui"
-                 "**/replaces_gui_resources.gui"
-                 "**/replaces_broken_gui_resources.gui"
-                 "**/fireworks_big.particlefx"
-                 "**/new.collisionobject"
-                 "**/three_shapes.collisionobject"
-                 "**/tile_map_collision_shape.collisionobject"
-                 "**/treasure_chest.model"
-                 "**/new.factory"
-                 "**/with_prototype.factory"
-                 "**/new.sound"
-                 "**/tink.sound"
-                 "**/new.camera"
-                 "**/non_default.camera"
-                 "**/new.tilemap"
-                 "**/with_cells.tilemap"
-                 "**/with_layers.tilemap"
-                 "**/test_attributes.material"
-                 "**/test.model"
-                 "**/empty_mesh.model"
-                 "**/test.label"
-                 "**/with_collection.collectionproxy"
-                 "**/ice.tilesource"
-                 "**/level.tilesource"
-                 "**/test.tileset"]]
-    (with-clean-system
-      (let [workspace (test-util/setup-workspace! world)
-            project (test-util/setup-project! workspace)
-            save-data-by-resource (into {}
-                                        (map (juxt :resource identity))
-                                        (project/all-save-data project))]
-        (doseq [query queries]
-          (testing (format "Saving %s" query)
-            (let [resource (ffirst (project/find-resources project query))
-                  resource-type (resource/resource-type resource)
-                  save-string (:content (save-data-by-resource resource))]
-              (if-some [read-fn (:read-fn resource-type)]
-
-                ;; We have a read-fn. Compare the read data representations.
-                (let [disk-pb-msg (read-fn resource)
-                      save-pb-msg (with-open [reader (StringReader. save-string)]
-                                    (read-fn reader))
-                      [only-in-disk-pb-msg only-in-save-pb-msg] (data/diff disk-pb-msg save-pb-msg)]
-                  (is (nil? only-in-disk-pb-msg))
-                  (is (nil? only-in-save-pb-msg))
-                  (when (or (some? only-in-disk-pb-msg)
-                            (some? only-in-save-pb-msg))
-                    (println "When comparing" (resource/proj-path resource))
-                    (prn 'disk only-in-disk-pb-msg)
-                    (prn 'save only-in-save-pb-msg)))
-
-                ;; We don't have a read-fn. Compare the strings.
-                (let [disk-string (slurp resource)]
-                  (is (= disk-string save-string)))))))))))
-
-(deftest save-all-literal-equality
```

Comment:
We have no replacement for this in the new save-data tests, because we're no longer interested in our saved files being compatible with the old editor. It used to serve some purpose in terms of covering some additional properties in the project files, but now we have full coverage without it. The content remains in the test project, since other tests interact with it, and we might as well ensure we can still load it. But you shouldn't need to ever update the file format of these legacy files anymore.